docs: add example tests and non-engineer intro copy (#259)
anti-work/shortest: packages/shortest/README.md

shortest([loginAsContractor, ...allAppActions]);
```

> Shortest allows non-engineers such as designers, marketers, and PMs to write tests. Here are some examples:

Reviewer: ```suggestion
Shortest's style allows non-engineers such as designers, marketers, and PMs to write tests. Here are some examples:
```

```typescript

> shortest("visit every page and ensure no typos")

Reviewer: ```suggestion
shortest("visit every page and ensure no typos");
```

> shortest("visit every page and ensure mobile layout isn't janky")

Reviewer: ```suggestion
shortest("visit every page and ensure mobile layout isn't janky");
```

> shortest("visit every page and ensure dark mode is considered")

Reviewer: ```suggestion
shortest("visit every page and ensure dark mode is considered");
```
